ovh/data_cloud_project_database.go

```diff
@@ -0,0 +1,205 @@
+				Required:    true,
+			},
+			"cluster_id": {
+				Type:        schema.TypeString,
+				Description: "Cluster ID",
+				Required:    true,
+			},
+
+			//Computed
+			"backup_time": {
+				Type:        schema.TypeString,
+				Description: "Time on which backups start every day",
+				Computed:    true,
+			},
+			"created_at": {
+				Type:        schema.TypeString,
+				Description: "Date of the creation of the cluster",
+				Computed:    true,
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Description: "Description of the cluster",
+				Computed:    true,
+			},
+			"endpoints": {
+				Type:        schema.TypeList,
+				Description: "List of all endpoints of the service",
+				Computed:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"component": {
+							Type:        schema.TypeString,
+							Description: "Type of component the URI relates to",
+							Computed:    true,
+						},
+						"domain": {
+							Type:        schema.TypeString,
+							Description: "Domain of the cluster",
+							Computed:    true,
+						},
+						"path": {
+							Type:        schema.TypeString,
+							Description: "Path of the endpoint",
+							Optional:    true,
+							Computed:    true,
+						},
+						"port": {
+							Type:        schema.TypeInt,
+							Description: "Connection port for the endpoint",
+							Optional:    true,
+							Computed:    true,
+						},
+						"scheme": {
+							Type:        schema.TypeString,
+							Description: "Scheme used to generate the URI",
+							Optional:    true,
+							Computed:    true,
+						},
+						"ssl": {
+							Type:        schema.TypeBool,
+							Description: "Defines whether the endpoint uses SSL",
+							Computed:    true,
+						},
+						"ssl_mode": {
+							Type:        schema.TypeString,
+							Description: "SSL mode used to connect to the service if the SSL is enabled",
+							Optional:    true,
+							Computed:    true,
+						},
+						"uri": {
+							Type:        schema.TypeString,
+							Description: "URI of the endpoint",
+							Optional:    true,
+							Computed:    true,
+						},
+					},
+				},
+			},
+			"flavor": {
+				Type:        schema.TypeString,
+				Description: "The node flavor used for this cluster",
+				Computed:    true,
+			},
+			"maintenance_time": {
+				Type:        schema.TypeString,
+				Description: "Time on which maintenances can start every day",
+				Computed:    true,
+			},
+			"network_type": {
+				Type:        schema.TypeString,
+				Description: "Type of network of the cluster",
+				Computed:    true,
+			},
+			"nodes": {
+				Type:        schema.TypeList,
+				Description: "List of nodes composing the service",
+				Computed:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"network_id": {
+							Type:        schema.TypeString,
+							Description: "Private network ID in which the node is",
+							Computed:    true,
```

Comment:
I think both network ID and subnet ID can be optionnal

Reply:
Optional field mean "a value can be given or not in configuration"
and a Optionnal / Computed field mean "a value can be given or not in configuration, if not it can be assigned or not during Create", computed field is optional is never sure to be set.

On other hand, after this comment, I remove some wrong `Optionnal` in fields with  "Optionnal / Computed" combo